tools/README.md

```diff
@@ -101,11 +100,20 @@ address = 0x9df878E37C4B851bbe44841e72299F6f317C685C
 password-file = ./keys/pass-UTC--2019-11-22T13-50-19.904075843Z--9df878e37c4b851bbe44841e72299f6f317c685c
 ```
 
-This script does not take into account fees, therefore it must the no-fee PFS
+This script does not take into account fees, therefore it must use the no-fee PFS
 (as per the example configuration above).
 
-pylint
-------
+Before using the stress test for the first time, you need to prepare the three nodes given in the config file:
+1. Create three accounts (e.g. with `geth account new`).
+2. Create the config file.
+3. Uncomment the `input()` line in `stress_test_transfers.py` and start the script with the config file. Now the three nodes are running and you can use other scripts to interact with them.
```

Comment:
I would suggest having a dummy config with the contents you have in here inside the repo and ask the user in this guide to just open the dummy config and fill in his values. And here just tell him to `./stress_test_transfers.py stress_test_transfers_config.ini`.